### PR TITLE
Remove unneeded json_query

### PR DESCRIPTION
**Pull request by pablintino**
As a pull request owner and reviewers, I checked that:
- [x] Appropriate testing is done and actually running
- [x] Appropriate documentation exists and/or is up-to-date

### CODE REVIEW

File: roles/repo_setup/tasks/artifacts.yml [hunk 1]
```diff
@@ -17,22 +17,22 @@
...
-    cifmw_repo_setup_full_hash: "{{ _hash['full_hash'] }}"
-    cifmw_repo_setup_commit_hash: "{{ _hash['commit_hash'] }}"
-    cifmw_repo_setup_distro_hash: "{{ _hash['distro_hash'] }}"
-    cifmw_repo_setup_extended_hash: "{{ _hash['extended_hash'] }}"
-    cifmw_repo_setup_dlrn_api_url: "{{ _hash['dlrn_api_url'] }}"
-    cifmw_repo_setup_release: "{{ _hash['release'] }}"
-    cacheable: true
+    - name: Export hashes facts for further use
+      ansible.builtin.set_fact:
+        cifmw_repo_setup_full_hash: "{{ _hash['full_hash'] }}"
```
Comment (cescgina) [inline roles/repo_setup/tasks/artifacts.yml:32]: this should be `_get_hash['full_hash']` or put back the `Store the needed data` task, if I'm reading correctly

File: hooks/playbooks/ceph-deploy.yml [hunk 2]
```diff
@@ -43,7 +43,13 @@
 
     - name: Extract ceph.conf from secret
       ansible.builtin.set_fact:
-        cifmw_ceph_conf_encoded: "{{ generated_uuid.stdout | from_json | json_query('data.\"ceph.conf\"') }}"
+        cifmw_ceph_conf_encoded: >-
+          {{
+            (
+              generated_uuid.stdout |
+              from_json
+            )['data']['ceph.conf']
```
Comment (viroel) [inline hooks/playbooks/ceph-deploy.yml:51]: the key has double quotes there, so I guess that it needs to be here too:
`\"ceph.conf\"`

Comment (viroel) [reply hooks/playbooks/ceph-deploy.yml:51]: podified-multinode-edpm-e2e-nobuild-tagged-crc-testing https://review.rdoproject.org/zuul/build/7310ea58970d4c129e23da44b275a061 : SUCCESS in 1h 06m 16s
and
https://logserver.rdoproject.org/34/51134/2/check/podified-multinode-edpm-e2e-nobuild-tagged-crc-testing/7310ea5/controller/ci-framework-data/artifacts/pre_deploy_ceph_deploy.yml

looks good, thanks for testing.

**Comment by viroel** [comment]
podified-multinode-edpm-e2e-nobuild-tagged-crc-testing https://review.rdoproject.org/zuul/build/7310ea58970d4c129e23da44b275a061 : SUCCESS in 1h 06m 16s
and
https://logserver.rdoproject.org/34/51134/2/check/podified-multinode-edpm-e2e-nobuild-tagged-crc-testing/7310ea5/controller/ci-framework-data/artifacts/pre_deploy_ceph_deploy.yml

looks good, thanks for testing.

**Review by rlandy — COMMENTED** [review]
/lgtm https://review.rdoproject.org/r/c/testproject/+/51134 passed

**Pull request merged**